### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by studiosnabeel**
Loom Video to show my code is working just kindly use npm start 
https://www.loom.com/share/b819ea49b0b146029dd82605eb86cad4

# Learning objectives:
- Use webpack to bundle JavaScript.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.

Highlights:
-Linter errors checked.
-Correct Github flow used.
-Work documented in a Professional way.
-Webpack installed correctly.
-Modular Javascript used.
-Javascript es6 used to add and remove tasks.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,103 @@
...
+document.addEventListener('DOMContentLoaded', () => {
+  UI.showTasks();
+  document.querySelectorAll('#para').forEach((paragraph, index) => {
+    paragraph.addEventListener('input', () => {
+      // const todo = new Todo(paragraph.textContent);
+      const Storage1 = JSON.parse(localStorage.getItem('tasks')) || [];
+      Storage1[index].item = paragraph.textContent;
+      localStorage.setItem('tasks', JSON.stringify(Storage1));
+      // console.log(Storage1[index].item);
+      // console.log(paragraph.textContent);
```
Comment (sajid-munawar) [inline src/index.js]: - [ ] Kindly remove commented codes from your code base. commented codes make your code look messy.

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,103 @@
...
+  });
+});
+
+// Add a task
+document.querySelector('.add-to-list').addEventListener('submit', (e) => {
+  // prevent the actual submit
+  e.preventDefault();
+
+  // get form values
+  const item = document.querySelector('#dataInput').value;
```
Comment (sajid-munawar) [inline src/index.js:70]: - [ ] Kindly make sure that while adding a new task to the todo list it should save to the localstorage and also it should appear on page. but now when I add new todo list task nothing happens even after cloning to my local machine and installing all dependencies.
https://www.loom.com/share/10ffc3d11f974a4393fb86a237c48e35

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
# Changes Required 
 ### Hi @studiosnabeel  
 Nice job, you are almost there.
 
 
 
 ## (Highlights) Good Points: 
 * Good commit messages. :heavy_check_mark:
 * No linter errors. :heavy_check_mark:
 * Followed Github flow :heavy_check_mark:
 * Professional README file. :heavy_check_mark:
 * PR has a good title and summary :heavy_check_mark:

 ## (Changes Required) Aspects to improve: 
 * [ ] Please check my inline comments.


 
 ## [Optional] suggestions:
 * Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag me in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 2 above)
Comment (studiosnabeel) [reply src/index.js:70]: @sajid-munawar Bro everything starts working after i write npm start on terminal but how can i make sure that everything works without needing npm start

Comment (studiosnabeel) [reply src/index.js:70]: @sajid-munawar Bro Kindly check in my Loom link everything is working and styled but i have to npm start on terminal to get everything to happen how can i make sure that i dont have to use npm ???? please guide me thankyou
https://www.loom.com/share/b819ea49b0b146029dd82605eb86cad4

File: src/index.js (hunk 1 above)
Comment (studiosnabeel) [reply src/index.js]: ok Thankyou i have removed all console log comments now

Comment (Abenezer-Tilahun) [reply src/index.js]: Fixed.

File: src/index.js (hunk 2 above)
Comment (Abenezer-Tilahun) [reply src/index.js:70]: Fixed.

File: modules/todo.js [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
-  constructor(desc, completed, id) {
-    this.desc = desc;
+// Class Todo
+export default class Todo {
+  constructor(item, completed = false, index) {
+    this.item = item;
+    this.index = index;
     this.completed = completed;
-    this.id = id;
   }
```
Comment (Abenezer-Tilahun) [inline modules/todo.js:7]: - [ Optional ] As stated the project requirement By default new tasks should have the property `index` set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). But In your Project the property `index` doesn't setup the specific `id` Kindly Adjust it.
 
Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @studiosnabeel     
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Implement the CRUD  functionality.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (sajid-munawar) [reply src/index.js:70]: thats cool everything is working fine.

**Pull request merged**